geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/InternalLocatorClusterManagementServiceIntegrationTest.java

```diff
@@ -56,13 +57,38 @@ public void tearDown() {
     }
   }
 
+  @Before
+  public void setup() throws URISyntaxException {
```

Comment:
I recommend moving all the field and var creations at the top, then move all the when blocks together, order both blocks in some way (I alphabetize), and then consider moving the subject(s) under test to the bottom of the setup method.

I also recommend cleaning up any statics in tearDown that you've configured in setUp. For example: how to undo this statement in tearDown:
```
BaseManagementService.setManagementService(cache, managementService);
```
The statics will still have this configured after each test completes.